statsmodels/genmod/generalized_linear_model.py

```diff
@@ -658,6 +658,37 @@ def estimate_scale(self, mu):
             raise ValueError("Scale %s with type %s not understood" %
                              (self.scaletype, type(self.scaletype)))
 
+    def tweedie_estimate_power(self, mu, low=1.01, high=5.):
```

Comment:
add a `method` argument after `mu`, even if it's currently not used.
I think we need to provide, eventually, alternatives to brentq.